packages/orbit-components/src/Breadcrumbs/index.tsx

```diff
@@ -18,6 +18,7 @@ const Breadcrumbs = ({
   spaceAfter,
   backHref,
   id,
+  ariaLabel = "Breadcrumbs",
```

Comment:
Why did you change it from singular to plural?

Reading this, they only use singular https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/examples/breadcrumb/
It's in singular also [here](https://www.airbnb.com/help/article/280).

Thinking about it, do we even need to open up the prop? Will anyone use anything else than "Breadcrumb"? Should they? 🤔

Reply:
Ok, I'll change plural to singular. 👍 

> Thinking about it, do we even need to open up the prop? Will anyone use anything else than "Breadcrumb"? Should they? 🤔

Yes, this announcement should be translated - I got inspired here: https://www.airbnb.com.ee/help/article/2855?_set_bev_on_new_domain=1742552838_EAYzc0ZTY1N2NhMW

Reply:
Ah, I see, OK then 👍